### ISSUE TITLE
libcalico yields misleading error from bad ETCD_AUTHORITY env var

### ISSUE THREAD

**Issue by ozdanborne** [projectcalico/libcalico#85]
If `ETCD_AUTHORITY` environment is missing a colon, pycalico will fail to initialize the `IPAMClient` with  a misleading stack trace. 

```
$ ETCD_AUTHORITY=erroneous python -c "import pycalico.ipam; pycalico.ipam.IPAMClient()"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/pycalico/datastore.py", line 137, in __init__
    (host, port) = etcd_authority.split(":", 1)
ValueError: need more than 1 value to unpack
```

[Link to line 137](https://github.com/projectcalico/libcalico/blob/master/calico_containers/pycalico/datastore.py#L137)

I propose that we should catch the value error and reword it to state "malformed ETCD_AUTHORITY", possibly in the form of a new MalformedEtcdAuthority exception?

**Comment by tomdee** [comment]
SGTM
